src/components/LocalizedDate.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+const LocalizedDate = ( {date} ) => {
+  const localizedDate = new Date(date).toLocaleDateString() 
+return (
+  <>
+  <strong>{localizedDate}</strong>&nbsp;
```

Comment:
let's remove the `&nbsp;`, it's not a requirement for the LocalizedDate component and it's only needed in the ClaimCardHeader